examples/webgl_materials_envmaps_ground-projected.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>threejs webgl - materials - hdr environment mapping</title>
+		<meta charset="utf-8" />
+		<meta
+			name="viewport"
+			content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0"
+		/>
+		<link type="text/css" rel="stylesheet" href="main.css" />
+	</head>
+	<body>
+		<div id="container"></div>
+		<div id="info">
+			<a href="https://threejs.org" target="_blank" rel="noopener">threejs</a> -
+			Ground projected environment mapping. Created by
+			<a href="https://twitter.com/CantBeFaraz" target="_blank" rel="noopener"
+				>Faraz Shaikh</a
+			>.
+		</div>
+
+		<!-- Import maps polyfill -->
+		<!-- Remove this when import maps will be widely supported -->
+		<script
+			async
+			src="https://unpkg.com/es-module-shims@1.3.6/dist/es-module-shims.js"
+		></script>
+
+		<script type="importmap">
+			{
+				"imports": {
+					"three": "../build/three.module.js"
+				}
+			}
+		</script>
+
+		<script type="module">
+			import * as THREE from "three";
+
+			import Stats from "./jsm/libs/stats.module.js";
+
+			import { GUI } from "./jsm/libs/lil-gui.module.min.js";
+			import { OrbitControls } from "./jsm/controls/OrbitControls.js";
+			import { HDRCubeTextureLoader } from "./jsm/loaders/HDRCubeTextureLoader.js";
+			import { RGBMLoader } from "./jsm/loaders/RGBMLoader.js";
+			import { DebugEnvironment } from "./jsm/environments/DebugEnvironment.js";
```

Comment:
It seems the imports of `HDRCubeTextureLoader`, `RGBMLoader` and `DebugEnvironment` can be removed.